templates/shared/combomarkdowneditor.tmpl

```diff
@@ -16,27 +16,27 @@ Template Attributes:
 	<div class="ui tab active" data-tab-panel="markdown-writer">
 		<markdown-toolbar class="gt-df">
 			<div class="markdown-toolbar-group">
-				<md-header class="markdown-toolbar-button">{{svg "octicon-heading"}}</md-header>
-				<md-bold class="markdown-toolbar-button">{{svg "octicon-bold"}}</md-bold>
-				<md-italic class="markdown-toolbar-button">{{svg "octicon-italic"}}</md-italic>
+				<md-header class="markdown-toolbar-button" data-tooltip-content="{{.locale.Tr "editor.header.tooltip"}}">{{svg "octicon-heading"}}</md-header>{{- /*Get rid of spaces between the buttons*/ -}}
```

Comment:
If you make the parent a flexbox (`gt-df`), the spacing between tags will not matter anymore.